<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose mode</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Play against AI</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_AI_easy">
      <property name="text">
       <string>Easy</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/ai_easy_icon.jpg</normaloff>icons/ai_easy_icon.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_AI_medium">
      <property name="text">
       <string>Medium</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/icons/ai_easy_icon.jpg</normaloff>:/new/prefix1/icons/ai_easy_icon.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_AI_hard">
      <property name="text">
       <string>Hard</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/icons/ai_easy_icon.jpg</normaloff>:/new/prefix1/icons/ai_easy_icon.jpg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Two players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_two_player">
      <property name="text">
       <string>click</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/icons/two_player_icon.png</normaloff>:/new/prefix1/icons/two_player_icon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_history">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>view history</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/history_icon.png</normaloff>:/new/prefix1/icons/history_icon.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
